dapr/clients/grpc/_response.py

```diff
@@ -744,3 +744,10 @@ def __init__(
     def status(self) -> TopicEventResponseStatus:
         """Gets the status."""
         return self._status
+
+
+class UnlockResponseStatus(Enum):
+    success = api_v1.UnlockResponse.Status.SUCCESS
+    lock_unexist = api_v1.UnlockResponse.Status.LOCK_UNEXIST
```

Comment:
ideally this should be LOCK_DOES_NOT_EXIST
@berndverst thoughts? I think this needs a change in the actual API ...

Reply:
Interesting for you to say this. The JS SDK took exactly the approach you mentioned with `LockStatus.LockDoesNotExist` (https://github.com/dapr/js-sdk/pull/306/files#diff-c35d0d1899e7ac2ed1bb0f34db296ec0d61ab48ed3ff970541eddeb6e5528bebR78) while the DotNet SDK kept the "unexist (`LockStatus.LockUnexist` in https://github.com/dapr/dotnet-sdk/pull/886/files#diff-865d34ecd04e0c417abc7b6f3fd425793190ebaed69638b8cb47f3c06df8f664R1441)

Reply:
Updated that. As for the actual API, that can be a follow up PR but it would have to be coordinated with JS and DotNet SDKs.